src/App.tsx

```diff
@@ -6,9 +6,12 @@ import { BentoProvider, Headline, Inset } from "@buildo/bento-design-system";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 import Home from "./pages/Home";
+import RestaurantDetail from "./pages/RestaurantDetail";
+import React from "react";
 
 function App() {
   const { t } = useTranslation();
+  const [restId, setRestId] = React.useState("0");
```

Comment:
The restaurant ID to display in the detail page should be better handled by using a URL parameter, instead of a state here.